.env.example

```diff
@@ -3,6 +3,7 @@ RIPPLED_WS_PORT=51233
 RIPPLED_RPC_PORT=51234
 RIPPLED_PEER_PORT=51235
 RIPPLED_HOST=your-rippled-host-url-here
+P2P_RIPPLED_HOST=your-rippled-host-that-can-run-admin-only-requests-url-here
```

Comment:
You should add a note here that s1/s2 are set up to make account_lines and gateway_balances admin-only (it's not reporting-specific)